Update api spec - Closes #419 (#426)
LiskHQ/lisk-docs: modules/ROOT/pages/reference/api.adoc

[[_overview]]
== Access Restrictions

The API endpoints are by default restricted to a whitelist of IPs that can be found in `config.json` in the section `api.access.whitelist`.
If you want your API to be accessable by the public, you can do this by changing `api.access.public` to `true`.
This will allow anyone to make requests to your Lisk Core node.
However some endpoints stay private, that means only a list of whitelisted IPs can successfully make API calls to that particular endpoint;
This includes all forging related API calls.

> By default, only the nodes' local IP is included in the whitelist, you can change the setting in `config.json` in the section `forging.access.whitelist`.

Reviewer: ```suggestion
By default, only the nodes' local IP is included in the whitelist, it is possible to change the setting in `config.json` in the section `forging.access.whitelist`.
```

Author: I will apply this now, but keep in mind the swagger spec is auto generated from the `swagger.yml` file of the Lisk SDK.

So to fix this for good, it needs to be fixed there.
